criu-3.15/lib/py/stack_map_utils.py

```diff
@@ -184,9 +187,10 @@ def print_stack_map_data(stack_maps):
                 al.print_val()
 
 
-def parse_stack_maps(section):
+def parse_stack_maps(section, arch = None):
```

Comment:
This file is very critical for the transformation code. Since you have changed the function definitions, can we please make sure that all the tests which normally pass also pass with these changes? Have you tested the code?

Reply:
This function change was considered for stack map record update on stack shuffle. The change has been tested for both crit recode and ss. Considering this review request would require me to obtain the stack map information differently and need testing both features again. How necessary is this change?

Reply:
I reviewed the changes, visually they don't seem to change the logic of the function but just collect some extra data. Running all recode tests successfully should be enough for merging the change.